dialer.go

```diff
@@ -20,20 +20,26 @@ import (
 	"crypto/rsa"
 	"crypto/tls"
 	_ "embed"
+	"encoding/binary"
+	"errors"
 	"fmt"
 	"net"
 	"strings"
 	"sync"
 	"sync/atomic"
 	"time"
 
-	alloydbadmin "cloud.google.com/go/alloydb/apiv1beta"
+	alloydbadmin "cloud.google.com/go/alloydb/apiv1alpha"
```

Comment:
`apiv1` should support this feature right?

Reply:
The v1 changes for go should be out now based on this release: https://github.com/googleapis/google-cloud-go/commit/b44c4b301a91e8d4d107be6056b49a8fbdac9003

Reply:
The changes aren't in a released version yet. So I bumped to beta. I'll send a separate PR to bump to v1 when the support is tagged in a release.